src/python/bcc/table.py

```diff
@@ -622,7 +622,7 @@ def _get_event_class(self):
         i = 0
         while i < num_fields:
             field = lib.bpf_perf_event_field(self.bpf.module, self._name, i).decode()
```

Comment:
looks like field has been decoded here?

Reply:
Sorry. My bad.
I thought I had checked against the current repo, but I was testing against Gentoo's stable...